code/modules/clothing/under/accessories.dm

```diff
@@ -1,230 +1,261 @@
-/obj/item/clothing/tie //Ties moved to neck slot items, but as there are still things like medals and armbands, this accessory system is being kept as-is

```

Comment:
They're both cr+lf. Is it because I renamed the file?
![crlf](https://cloud.githubusercontent.com/assets/9963278/26810173/d07a7eee-4a1e-11e7-913e-043e2e895e97.PNG)